# Miscelaneous aiding status
# custom misc_aiding_status_t
	uint8 zero_velocity_x  # 0 = Source Disabled | 1 = Invalid | 2 = Rejected | 3 = Successfully Processed
	uint8 zero_velocity_y  # 0 = Source Disabled | 1 = Invalid | 2 = Rejected | 3 = Successfully Processed
	uint8 zero_velocity_z  # 0 = Source Disabled | 1 = Invalid | 2 = Rejected | 3 = Successfully Processed
	uint8 magnetic_heading  # 0 = Source Disabled | 1 = Invalid | 2 = Rejected | 3 = Successfully Processed
	uint8 zero_heading_change  # 0 = Source Disabled | 1 = Invalid | 2 = Rejected | 3 = Successfully Processed
	uint8 barometric_altitude  # 0 = Source Disabled | 1 = Invalid | 2 = Rejected | 3 = Successfully Processed
